samples/notebooks/prediction/SDK_Custom_Preprocess.ipynb

```diff
@@ -0,0 +1,1321 @@
+   "metadata": {
+    "id": "BF1j6f9HApxa"
+   },
+   "source": [
+    "### Set up your Google Cloud project\n",
+    "\n",
+    "**The following steps are required, regardless of your notebook environment.**\n",
+    "\n",
+    "1. [Select or create a Google Cloud project](https://console.cloud.google.com/cloud-resource-manager). When you first create an account, you get a $300 free credit towards your compute/storage costs.\n",
+    "\n",
+    "1. [Make sure that billing is enabled for your project](https://cloud.google.com/billing/docs/how-to/modify-project).\n",
+    "\n",
+    "1. [Enable the Vertex AI API and Compute Engine API](https://console.cloud.google.com/flows/enableapi?apiid=aiplatform.googleapis.com,compute_component).\n",
+    "\n",
+    "1. If you are running this notebook locally, you will need to install the [Cloud SDK](https://cloud.google.com/sdk).\n",
+    "\n",
+    "1. Enter your project ID in the cell below. Then run the cell to make sure the\n",
+    "Cloud SDK uses the right project for all the commands in this notebook.\n",
+    "\n",
+    "**Note**: Jupyter runs lines prefixed with `!` or `%` as shell commands, and it interpolates Python variables with `$` or `{}` into these commands."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "WReHDGG5g0XY"
+   },
+   "source": [
+    "#### Set your project ID\n",
+    "\n",
+    "**If you don't know your project ID**, you may be able to get your project ID using `gcloud`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "oM1iC_MfAts1"
+   },
+   "outputs": [],
+   "source": [
+    "# Get your Google Cloud project ID from gcloud\n",
+    "shell_output=!gcloud config list --format 'value(core.project)' 2>/dev/null\n",
+    "\n",
+    "try:\n",
+    "    PROJECT_ID = shell_output[0]\n",
+    "except IndexError:\n",
+    "    PROJECT_ID = None\n",
+    "\n",
+    "print(\"Project ID:\", PROJECT_ID)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "qJYoRfYng0XZ"
+   },
+   "source": [
+    "Otherwise, set your project ID here."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "riG_qUokg0XZ"
+   },
+   "outputs": [],
+   "source": [
+    "if PROJECT_ID == \"\" or PROJECT_ID is None:\n",
+    "    PROJECT_ID = \"[PROJ_ID]\"  # @param {type:\"string\"}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "XoEqT2Y4DJmf"
+   },
+   "source": [
+    "### Configure project and resource names"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "MzGDU7TWdts_"
+   },
+   "outputs": [],
+   "source": [
+    "REGION = \"us-central1\"  # @param {type:\"string\"}\n",
+    "MODEL_ARTIFACT_DIR = \"sklearn-cpr-preprocess-model\"  # @param {type:\"string\"}\n",
+    "REPOSITORY = \"kathytang-custom-preprocess-container-prediction\"  # @param {type:\"string\"}\n",
```

Comment:
Repository, image name and model display name.

Reply:
Good catch, I always forget to change these back..